pkgs/development/tools/goda/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "goda";
+  version = "0.5.1";
+
+  src = fetchFromGitHub {
+    owner = "loov";
+    repo = "goda";
+    rev = "v${version}";
+    sha256 = "sha256-tXXUDeqGjv6T2eI/VJ+kwPKJLT7D1nO9BaKN5FHS34I=";
+  };
+
+  vendorSha256 = "sha256-OyQEw6mRrRneo3T8wns0doU4lxJYEoilTd30xctLBJ4=";
+
```

Comment:
You can add:
```
  passthru.updateScript = nix-update-script {
    attrPath = pname;
  };
```
for automated future updates.

Reply:
Thanks for the suggestion! Done.

Reply:
You'll need `nix-update-script` in the arguments as well.